### PR TITLE
[ADD] partner_second_lastname addon

### PR DESCRIPTION
**Pull request by antespi**
This module adds a second lastname optional field for partners

### CODE REVIEW

**Comment by antespi** [comment]
@eLBati this addon changes firtsname behavior, then firstname tests will fail always, broking Travis test in all future PRs after merged.
What can I do? Publish this addons in a non-OCA repo?

@pedrobaeza what do you think? other opinions?

Thanks for feedback

**Comment by eLBati** [comment]
@antespi I don't understand: I see travis green here

**Comment by antespi** [comment]
@eLBati that's right, good then. 

We have this addon in production in one customer working well. Could you review for merging?. 
Thanks in advance

File: partner_second_lastname/models/res_partner.py [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+class ResPartner(models.Model):
+    _inherit = 'res.partner'
+
+    lastname_second = fields.Char(string='Second lastname')
+
+    @api.one
+    @api.depends('firstname', 'lastname', 'lastname_second', 'company_id')
+    def _compute_name(self):
+        names = (self.lastname, self.lastname_second, self.firstname)
+        if self.company_id.names_order == 'first_last':
```
Comment (eLBati) [inline partner_second_lastname/models/res_partner.py:38]: Where is `names_order` defined?

Comment (antespi) [inline partner_second_lastname/models/res_partner.py:38]: This featured is merge pending. We have to wait to PRs:
- https://github.com/OCA/partner-contact/pull/139
- https://github.com/OCA/partner-contact/pull/140

**Comment by yajo** [comment]
Yikes! I have a module for this too waiting for #139.

**Comment by yajo** [comment]
It's here, FWIW: https://github.com/grupoesoc/partner-contact/tree/partner_lastname2/partner_lastname2. I will not PR until #139 is merged.

File: partner_second_lastname/static/description/icon.svg [hunk 2]
```diff
@@ -0,0 +1,79 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```
Comment (yvaucher) [inline partner_second_lastname/static/description/icon.svg:1]: Using default OCA icon you don't need to add the svg we already have it in another repo

File: partner_second_lastname/tests/__init__.py [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+# of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA  02111-1307, USA.
```
Comment (yvaucher) [inline partner_second_lastname/tests/__init__.py:27]: This header is GPL 2 is it intended?

File: partner_second_lastname/tests/base.py [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA  02111-1307, USA.
+
```
Comment (yvaucher) [inline partner_second_lastname/tests/base.py:27]: You might want to change this header to AGPL

File: partner_second_lastname/views/res_partner_view.xml [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+        </field>
+        <xpath expr="//field[@name='child_ids']/form//field[@name='lastname']"
+               position="after">
+            <field name="lastname_second"/>
+        </xpath>
+    </field>
+</record>
+
+</data>
+</openerp>
```
Comment (yvaucher) [inline partner_second_lastname/views/res_partner_view.xml:32]: Please use POSIX file format with line break on last line

**Comment by rafaelbn** [comment]
@yvaucher this PR will be closed. It were a PR here https://github.com/OCA/partner-contact/pull/105 and here https://github.com/OCA/partner-contact/pull/153 with same functionality. Could you review https://github.com/OCA/partner-contact/pull/153 please? I'm waiting @antespi  review https://github.com/OCA/partner-contact/pull/153 to close this one.

**Comment by antespi** [comment]
I close this PR in favor to https://github.com/OCA/partner-contact/pull/153.
I've some improvements pending. I'll propuse them in a separate PR after this PR were merged:
- https://github.com/OCA/partner-contact/pull/153
- https://github.com/OCA/partner-contact/pull/159

@yvaucher, thanks for your comments, I'll take care in my future PRs
@eLBati , thanks for your review. I'll add names_order feature after https://github.com/OCA/partner-contact/pull/159

**Pull request closed without merging**